README.md

```diff
@@ -1,11 +1,28 @@
 
-# api-gatekeeper-frontend
+# API Gatekeeper Frontend
 
 [![Build Status](https://travis-ci.org/hmrc/api-gatekeeper-frontend.svg?branch=master)](https://travis-ci.org/hmrc/api-gatekeeper-frontend) [ ![Download](https://api.bintray.com/packages/hmrc/releases/api-gatekeeper-frontend/images/download.svg) ](https://bintray.com/hmrc/releases/api-gatekeeper-frontend/_latestVersion)
 
-This is a placeholder README.md for a new repository
+This is a Play Framework frontend application for HMRC's internal users.
```

Comment:
I think a really good example of this is the nisp frontend: https://github.com/hmrc/nisp-frontend